apps/hyd/BUILD

```diff
@@ -0,0 +1,83 @@
+load("@aspect_rules_js//js:defs.bzl", "js_library")
+load("@aspect_rules_ts//ts:defs.bzl", "ts_config")
+load("@bazel_skylib//rules:build_test.bzl", "build_test")
+load("@npm//:defs.bzl", "npm_link_all_packages")
+load("@npm//apps/hyd:next/package_json.bzl", next_bin = "bin")
```

Comment:
You would not BELIEVE how huge of a rabbit hole using `@npm//blog:next` copy pasta here was. Terrible, terrible errors